components/Coin.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import MaterialIcon from 'react-native-vector-icons/MaterialIcons';
+
+MaterialIcon.loadFont();
+
+const Coin: FC = ({ crypto }: any) => {
+    return (
+        <View style={styles.cryptoContainer}>
+            <View style={{ flexDirection: 'row' }}>
+                <Image
+                    source={crypto.image}
+                    style={styles.cryptoImage} />
+                <View>
+                    <Text style={styles.cryptoTextName}>{crypto.name}</Text>
+                    <Text style={styles.cryptoTextInitials}>{crypto.initials}</Text>
+                </View>
+            </View>
+
+            <View style={styles.cryptoValuesContainer}>
+                <Text style={styles.cryptoTextName}>$ {crypto.value}</Text>
+                <View style={{ flexDirection: 'row' }}>
+                    <MaterialIcon name={crypto.icon} size={17} color={crypto.color} />
+                    <Text style={{ color: (crypto.color) }}>{crypto.metric} %</Text>
+                </View>
+            </View>
+
+        </View>
+    );
+};
+
+export default Coin;
+
+const styles = StyleSheet.create({
```

Comment:
[Minor] Los styles deberian in en su propio archivo, no en el mismo que el componente

Reply:
por ejemplo en este caso tendria la siguiente estructura 

components/Coin/index.tsx
components/Coin/styles.ts

renombrando el actual Coin.tsx y guardándolo en una carpeta del mismo nombre